### PR TITLE
(WIP ) Added getNewPlugins Function

### PR DESCRIPTION
**Pull request by sladyn98**
This feature deals with the display of new plugins.

### CODE REVIEW

**Comment by LinuxSuRen** [comment]
The test cases are needed once you done with your work. And thanks for your contribution.

File: client/pluginApi.go [hunk 1]
```diff
@@ -172,3 +172,23 @@ func (d *PluginAPI) collectDependencies(pluginName string) (plugins []PluginInfo
 	}
 	return
 }
+
+func (d *PluginAPI) NewPlugins()(plugin *PluginInfo, err error) {
```
[the remark this answers is not in the document]
Comment (sladyn98) [reply client/pluginApi.go]: Yep working on it :)

[the remark this answers is not in the document]
Comment (LinuxSuRen) [reply client/pluginApi.go]: Cool!

**Comment by sladyn98** [comment]
firstRelease": "2019-09-05T01:58:49.53Z",
            "buildDate": "2019-09-05",
            "categories": [],
            "dependencies": [
                {
                    "name": "jackson2-api",
                    "title": "Jackson 2 API",
                    "optional": false,
                    "version": "2.9.9",
                    "implied": false
                },
                {
                    "name": "trilead-api",
                    "title": "Trilead API",
                    "optional": false,
                    "version": "1.0.4",
                    "implied": true
                }
            ],
            "maintainers": [
                {
                    "id": "croninka",
                    "name": null,
                    "email": null
                }
            ],
            "excerpt": "Kubernetes Client API plugin for use by other Jenkins plugins.",
            "gav": "org.jenkins-ci.plugins:kubernetes-client-api:1.0.0",
            "labels": [],
            "name": "kubernetes-client-api",
            "previousTimestamp": null,
            "previousVersion": null,
            "releaseTimestamp": "2019-09-05T01:58:49.00Z",
            "requiredCore": "2.176.1",
            "scm": {
                "issues": null,
                "link": "https://github.com/jenkinsci/kubernetes-client-api-plugin",
                "inLatestRelease": null,
                "sinceLatestRelease": null,
                "pullRequests": null
            },
            "sha1": "g7jjcEUD/rEmEZaiOPbPB9QeYII=",
            "stats": {
                "installations": null,
                "installationsPercentage": null,
                "installationsPerVersion": null,
                "installationsPercentagePerVersion": null,
                "currentInstalls": 0,
                "trend": 0
            },
            "title": "Kubernetes Client API",
            "url": "http://updates.jenkins-ci.org/download/plugins/kubernetes-client-api/1.0.0/kubernetes-client-api.hpi",
            "version": "1.0.0",
            "securityWarnings": null,
            "wiki": {
                "content": null,
                "url": "https://wiki.jenkins.io/display/JENKINS/Kubernetes+Client+API+Plugin"
            }
        },

we might need a new struct to unmarshall this data @LinuxSuRen

**Comment by LinuxSuRen** [comment]
@sladyn98 Sure, please go ahead.

**Comment by LinuxSuRen** [comment]
@sladyn98 Hi there, are you still interested in the current PR? It's been for a while.

**Comment by sladyn98** [comment]
@LinuxSuRen Yeah I will be working on this, was a bit busy lately.

**Comment by LinuxSuRen** [comment]
ping @sladyn98

**Comment by sladyn98** [comment]
@LinuxSuRen can't load package: package github.com/sladyn98/Jenkins/jenkins-cli/client: module github.com/sladyn98/Jenkins@latest found (v0.0.0-20190604142804-fdcaef37b9f6), but does not contain package github.com/sladyn98/Jenkins/jenkins-cli/client
When I run a test I am getting this error ?

**Comment by LinuxSuRen** [comment]
what's the version of go?

**Comment by sladyn98** [comment]
go version go1.13.5 linux/amd64

**Comment by LinuxSuRen** [comment]
I suggest that you sync with master branch. There're a lot of changes happen.

**Comment by sladyn98** [comment]
Will reopen this soon

**Pull request closed without merging**